.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode7.3
+osx_image: xcode8
```

Comment:
Currently, this PR breaks compatibility with Xcode 7 by the diff below.
https://github.com/kylef/WebLinking.swift/pull/6/files#diff-6f2529f6d403a440ce762df1db755907R102

We can support both versions of Xcode if we add `#if swift(>=2.3)` there.
I'll work on it.